src/js/README.md

```diff
@@ -13,44 +13,26 @@ This is preview version, do not use it in production!
 
 ### Build Bindings
 
-#### Build OpenVINO GenAI as OpenVINO Extra Module
-
-OpenVINO GenAI Node.js bindings can be built as an extra module during the OpenVINO build process. This method simplifies the build process by integrating OpenVINO GenAI directly into the OpenVINO build.
-
-1. Clone OpenVINO repository:
-   ```sh
-   git clone --recursive https://github.com/openvinotoolkit/openvino.git
-   ```
-1. Configure CMake with OpenVINO extra modules:
-   ```sh
-   cmake -DOPENVINO_EXTRA_MODULES=*path to genai repository directory* -DCPACK_ARCHIVE_COMPONENT_INSTALL=OFF \
-         -DCPACK_GENERATOR=NPM \
-         -DENABLE_PYTHON=OFF \
-         -DENABLE_WHEEL=OFF \
-         -DCPACK_PACKAGE_FILE_NAME=genai_nodejs_bindings \
-         -S ./openvino -B ./build
-   ```
-1. Build OpenVINO archive with GenAI:
-   ```sh
-   cmake --build ./build --target package -j
-   ```
-
-1. Put Node.js bindings into npm package `bin` directory and install dependencies:
-   ```sh
-   mkdir ./src/js/bin/
-   tar -xvf ./build/genai_nodejs_bindings.tar.gz --directory ./src/js/bin/
-   cd ./src/js/
-   npm install
-   ```
-1. Run tests to be sure that everything works:
-   ```sh
-   npm test
-   ```
-
-### Using as npm Dependency
+Build OpenVINO™ GenAI JavaScript Bindings from sources following the [instructions](../js/BUILD.md).
+
+### Using the package from your project
+
+#### Option 1 - using npm:
 
 To use this package locally use `npm link` in `src/js/` directory
 and `npm link genai-node` in the folder where you want to add this package as a dependency
 
-To extract this package and use it as distributed npm package run `npm package`.
-This command creates archive that you may use in your projects.
+#### Option 2 - using package.json:
+
+Add the `genai-node` package manually by specifying the path to the `src/js/` directory in your `package.json`:
+
+```
+"genai-node": "file:*path-to-current-directory*"
+```
+
+### Installation verification
+
+Run a simple NodeJS script:
+```nodejs
+const { Pipeline } = require('genai-node');
+```
```

Comment:
We didn't think about it when we were writing `BUILD.md`, but maybe you can provide a one-liner like `python -c "import openvino_genai; print(openvino_genai.__version__)"` for JS. That would simplify verification

Reply:
https://github.com/openvinotoolkit/openvino.genai/pull/1739

Reply:
Sure, we can